src/pages/resultsView/survival/SurvivalTab.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
+import {Chart} from 'chart.js';
+import SurvivalChart from "./SurvivalChart";
+import {ResultsViewPageStore} from "../ResultsViewPageStore";
+import Loader from "../../../shared/components/loadingIndicator/LoadingIndicator";
+import {observer} from "mobx-react";
+import styles from "./styles.module.scss";
+
+export interface ISurvivalTabProps {
+    store: ResultsViewPageStore
+}
+
+export function hideEventPoints(chart: any): void {
+
+    chart.config.data.datasets.map((dataset: any) => {
+        let previousData: number;
+        dataset.data.map((data: any, index: number) => {
+            if (previousData && data.y !== previousData) {
+                let meta = dataset._meta[0] == null ? dataset._meta[1] : dataset._meta[0];
+                meta.data[index]._model.borderColor = 'rgba(0, 0, 0, 0)';
+            }
+            previousData = data.y;
+        });
+    });
+}
+
+@observer
+export default class SurvivalTab extends React.Component<ISurvivalTabProps, {}> {
+
+    private overallSurvivalTitleText = 'Overall Survival Kaplan-Meier Estimate';
+    private diseaseFreeSurvivalTitleText = 'Disease Free Survival Kaplan-Meier Estimate';
+
+    componentWillMount() {
+        Chart.pluginService.register({
```

Comment:
we can take this out no?

Reply:
Yes, will do later, we can merge for now.